Makefile

```diff
@@ -19,7 +19,7 @@ ethereum-build: ethereum-clean
 	@cd ./contracts/solidity/ && forge build
 
 ethereum-test: ethereum-clean
-	@cd ./contracts/solidity/ && forge test
+	@cd ./contracts/solidity/ && forge test -vvvvv
```

Comment:
I don't think the `-vvvv` is necessary for this target. If the developers are debugging they can use the `forge test -vvvv` by their own